bundle/src/test/java/com/adobe/acs/commons/properties/TemplatedDialogTestUtil.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * #%L
+ * ACS AEM Commons Bundle
+ * %%
+ * Copyright (C) 2019 Adobe
```

Comment:
nit: all of the copyrights on all the new files should probably be 2021 since it's...well...2021 now :D

Reply:
Updated license years (man this code has lived a long time)